CRM/Core/Smarty/plugins/block.localize.php

```diff
@@ -26,11 +26,19 @@
  *   {ts} block contents from the template.
  * @param CRM_Core_Smarty $smarty
  *   The Smarty object.
+ * @param bool $repeat
+ *   Confusing variable that means it's either the opening tag or you can use
+ *   it to signal back not to repeat.
  *
  * @return string
  *   multilingualized query
  */
-function smarty_block_localize($params, $text, &$smarty) {
+function smarty_block_localize($params, $text, &$smarty, &$repeat) {
```

Comment:
The pass-by-ref seems unnecessary

Reply:
True at the moment it's not used that way, just that's the official signature.